app/models/package_manager/go.rb

```diff
@@ -53,34 +53,55 @@ def self.update(name)
       # example: github.com/myexample/modulename/v2
       if (matches = name.match(VERSION_MODULE_REGEX))
         PackageManagerDownloadWorker.perform_async(self.name, matches[1])
```

Comment:
We could remove this call to update the base module from here, but it may help fill in some missing data for now.

Reply:
Yea it's probably not necessary? Esp if we go back and do some cleanup.

Reply:
I think I will keep it in here for now and then remove it once I figure out how to get all the base modules into Libraries.